src/components/change-requests/review-change-request/review-change-request/review-change-request.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { Button, Form } from 'react-bootstrap';
+import { useHistory } from 'react-router-dom';
+import {
+  ChangeRequest,
+  StandardChangeRequest,
+  ActivationChangeRequest,
+  StageGateChangeRequest,
+  ChangeRequestType,
+  ChangeRequestExplanation
+} from 'utils';
+import PageBlock from "../../../shared/page-block/page-block";
+import PageTitle from "../../../shared/page-title/page-title";
+
+interface ReviewChangeRequestViewProps {
+  crId: number,
+  accepted: boolean
+}
+
+const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
+  crId,
+  accepted
+}: ReviewChangeRequestViewProps) => {
+  const history = useHistory();
+
+  const handleConfirm = () => {
+    // temporary redirect to change requests table
```

Comment:
I actually like this because the "better" way would be a tad more complicated than I think is worth our effort given how close the launch deadline is.

Just for your knowledge tho, the "better" way would be to redirect the user back to the detail page for the change request and display a toast or a banner saying "canceled" or "confirmed", and potentially a loading spinning while it's still loading or something, but that can be done later as part of UX improvement efforts when we're not itching to launch